src/resources/help/en_US/relnotes3.1.1.html

```diff
@@ -25,6 +25,7 @@
 <p><b><font size="4">Bugs fixed in version 3.1.1</font></b></p>
 <ul>
   <li>Corrected a bug where certain types of mili datasets would cause unexpected crashes.</li>
+  <li>Corrected a bug where the DLC wizard would always enter zero for the simulation cycle.</li>
```

Comment:
Please change "DLC" to "<i>Data level comparison</i>".

Reply:
Fixed with last commit.

Reply:
On second thought, can I even merge this into 3.1.1 because of the GUI change? I'm not sure if this even counts as a GUI change...

Reply:
Nothing changed in the interface, you just fixed a bug, so, yes, you can put it on the 3.1RC.